index.php

```diff
@@ -4,10 +4,13 @@
 	require_once 'server/header.php';
 ?>
 	<body>
-	
-		<script src="assets/js/main.js"></script>
+		<div>
+			VITA Home Page
+		</div>
+		<!-- TODO -->
```

Comment:
I just put some random text on the page so we could see the different pages when you click between them in the nav bar. It was more just a comment that it won't say "VITA Home Page". And I guess just the fact that the whole page is a TODO. lol